assets/js/blocks/cart-checkout-shared/block-settings/index.tsx

```diff
@@ -0,0 +1,70 @@
+/**
+ * External dependencies
+ */
+import { InspectorControls } from '@wordpress/block-editor';
+import { PanelBody, ToggleControl } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+import { CartCheckoutFeedbackPrompt } from '@woocommerce/editor-components/feedback-prompt';
+import { addFilter, hasFilter } from '@wordpress/hooks';
+import { createHigherOrderComponent } from '@wordpress/compose';
+
+const BlockSettings = ( { attributes, setAttributes } ) => {
+	const { hasDarkControls } = attributes;
+	return (
+		<InspectorControls>
+			<PanelBody title={ __( 'Style', 'woo-gutenberg-products-block' ) }>
+				<ToggleControl
+					label={ __(
+						'Dark mode inputs',
+						'woo-gutenberg-products-block'
+					) }
+					help={ __(
+						'Inputs styled specifically for use on dark background colors.',
+						'woo-gutenberg-products-block'
+					) }
+					checked={ hasDarkControls }
+					onChange={ () =>
+						setAttributes( {
+							hasDarkControls: ! hasDarkControls,
+						} )
+					}
+				/>
+			</PanelBody>
+			<CartCheckoutFeedbackPrompt />
```

Comment:
@opr, previously, when selecting the `Cart` or `Checkout` block, we could see the `feedback` notice. But, for some reason, this doesn't show up anymore!

<img src="https://user-images.githubusercontent.com/14235870/184043271-faf9a571-8fe6-40c5-bc70-42ad67b68c2f.png" alt="image" width="300"/>